backend/src/db/dbAccessor.ts

```diff
@@ -0,0 +1,38 @@
+import {Database} from "sqlite3";
+
+const sqlite3 = require('sqlite3').verbose();
```

Comment:
require und import wenns geht nicht vermischen. Wir benutzen im backend import, daher müsste das hier in 2 Zeilen aufgeteilt werden.

Reply:
Hat ESLint jetzt auch angemeckert :D

Reply:
und ich habs entsprechend angepasst :D